### ISSUE TITLE
Router health checks failed

### ISSUE THREAD

**Issue by 0sorkon** [apache/cloudstack#8569]
##### ISSUE TYPE

 * Other

##### COMPONENT NAME

~~~
VR
~~~

##### CLOUDSTACK VERSION

~~~
CloudStack 4.18.1.0
~~~

##### CONFIGURATION

n/a

##### OS / ENVIRONMENT

n/a

##### SUMMARY

Sometimes dns_check.py error occurs after creating/deleting/renaming VMs:  Missing entries for VMs in /etc/hosts. What is the correct way to fix it? Why it occurs and how to avoid it in the future

##### STEPS TO REPRODUCE

~~~

~~~

##### EXPECTED RESULTS

~~~

~~~

##### ACTUAL RESULTS

~~~

~~~

**Comment by weizhouapache** [comment]
@0sorkon 
this is probably same as #7653 
was the VR created from 4.18.1 systemvm template, or a older systemvm template ?
patch the VR, or restart network with cleanup, if the VR is old.

**Comment by DaanHoogland** [comment]
@0sorkon please let us know if @weizhouapache 's advice helped.

**Comment by 0sorkon** [comment]
No, VR is v4.18.1.0. And cloudstack 4.18.1 on all kvm hosts and management servers.

**Comment by weizhouapache** [comment]
> No, VR is v4.18.1.0. And cloudstack 4.18.1 on all kvm hosts and management servers.

@0sorkon 
are you able to reproduce the issue ? if yes, can you share the steps ?

**Comment by 0sorkon** [comment]
Yes, I can reproduce that error. Unfortunately, I cannot determine what conditions are necessary for it to occur. Just now I just created some vm with cloudmonkey and the error appeared again. No special actions. And I don't see any error messages in the /var/log/cloud.log and /var/log/messages logs on VR. Perhaps this should be looked for somewhere else?

**Comment by DaanHoogland** [comment]
@0sorkon , just a wild guess;
can you check your network topology?
-  what address reanges are being used in your setup?
-  no conflicts?
I have no idea what might be wrong but this is just a first level check.

**Comment by weizhouapache** [comment]
closing as it is same as #8761
